CMSgov/mint-app
# [EASI-2236] FE work for Model Tab View and Table

# EASI-2236

## Changes and Description

- Added a view for Models that is the component for /models to match figma
- Added new query `GetAllModelPlans` and its types
- Added read only model plan i18n file
- Created new Read Only Table 
- Moved ModelPlan from `src/views/ModelPlan/index.tsx` to its own folder `src/views/ModelPlan/ModelPlanOverview/index.tsx`



## How to test this change

1. Start up app with pre-seeded data
2. Click on `Models` on navigation bar
3. Check to see the page matches [Figma](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?node-id=753%3A70533)
4. Check to see filtering works
    1. Type in `Comp` and the filtering actually works
5. Clicking link will send you to the task list 
6. Check to see correct data is displayed in the table (`Category`, `Status`, `Model start date`) 
    1. CRs and TDLs have not been implemented yet, so it currently says `To be determined`


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
